src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -241,6 +241,13 @@ public boolean getSensorOverride() {
 		return sensorOverride.getBoolean(false);
 	}
 
+	public boolean isIntakeOn() {
+		return (intakeMotorFront.get() != 0
```

Comment:
Just want to verify this logic is correct.  I may be wrong (I am assuming we are giving drive team feedback that intake can intake a note), but seems like if intake is in reject mode (assuming < 0), that we shouldn't be lighting up the intake on signal.

Reply:
Drive team told me to leave rejecting to be yellow as well.